test/unpooling-2d.cc

```diff
@@ -222,3 +222,84 @@ TEST_F(Unpooling2DTestX32, matches_operator_api)
 
   ASSERT_EQ(subgraph_output, operator_output);
 }
+
+TEST_F(Unpooling2DTestX32, reshape_output)
+{
+  ASSERT_EQ(xnn_status_success, xnn_initialize(/*allocator=*/nullptr));
+
+  // Call subgraph API.
+  xnn_subgraph_t subgraph = nullptr;
+  ASSERT_EQ(xnn_status_success, xnn_create_subgraph(2, /*flags=*/0, &subgraph));
+  std::unique_ptr<xnn_subgraph, decltype(&xnn_delete_subgraph)> auto_subgraph(subgraph, xnn_delete_subgraph);
+
+  uint32_t input_value_id = XNN_INVALID_NODE_ID;
+  ASSERT_EQ(
+    xnn_status_success, xnn_define_tensor_value(
+                          subgraph, xnn_datatype_fp32, input_value_dims.size(), input_value_dims.data(), nullptr,
```

Comment:
wrong amount of indent.  either a long line or indent by 2

Reply:
hmm i see, I'm just keeping consistent with the tests above, but let me know if you want to fix all of those in this PR.